apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/TableRegistryInitializer.java

```diff
@@ -135,12 +133,15 @@
     private TradeTable tradeTable;
     @Inject
     private AccountControlPhasingTable accountControlPhasingTable;
+    @Inject
+    private PollTable pollTable;
```

Comment:
In general that is not needed. The main purpose of that class was not make early init by using `XXX.init();`
When we removed all XXX.init(); then we can delete TableRegistryInitializer.java

So you can remove such @Inject and check it works fine.

Reply:
It's out of the scope of this PR. We added beans to TableRegistryInitializer because of problems with time init in Weld. So it should be investigated in a separate PR